base/Dockerfile

```diff
@@ -40,4 +40,9 @@ RUN chmod 755 /usr/local/bin/go-wrapper
 RUN ln -s /go/src/app /app
 WORKDIR /go/src/app
 
+ENV CLOUD_DEBUG_ENABLE "true"
```

Comment:
I added this at first thinking that users who are not using AE Flex or GCE can also use this image. Should we add an env to let user choose if s/he wants to turn it off?